src/core/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 set(_srcs
-    charpak_mod.F90
+    precision_mod.F90
     constants.F90
+    charpak_mod.F90
     diagnostic_mod.F90
     error_mod.F90
     precision_mod.F90
     species_mod.F90
-    qfyaml_mod.F90)
+    qfyaml_mod.F90
+    init_mod.F90
+    test_main.F90)
```

Comment:
This program (`test_main`) isn't part of the library. We can move to a separate `tests` directory.

Reply:
@zmoon agree just building things out right now.  Constantly cleaning up and adding

Reply:
sure sure but the sooner we set it up, easier to add more

Reply:
started moving it